src/views/bootablevolumes/list/hooks/useBootableVolumesFilters.ts

```
@@ -26,19 +28,17 @@ import { RowFilter } from '@openshift-console/dynamic-plugin-sdk';
 import { BootableResource } from '../../utils/types';
 import { getPreferenceOSType } from '../../utils/utils';
 
-const useBootableVolumesFilters = (): RowFilter<BootableResource>[] => {
+const useBootableVolumesFilters = (bootableVolumes: BootableVolume[]): {filtersWithSelect: RowFilter<BootableResource>[], rowFilters: RowFilter<BootableResource>[]} => {
```

ESLint

[error] Replace `bootableVolumes:·BootableVolume[]):·{filtersWithSelect:·RowFilter<BootableResource>[],·rowFilters:·RowFilter<BootableResource>[]` with `⏎··bootableVolumes:·BootableVolume[],⏎):·{⏎··filtersWithSelect:·RowFilter<BootableResource>[];⏎··rowFilters:·RowFilter<BootableResource>[];⏎`

(prettier/prettier)